### PR TITLE
Feat-implemented-redux-for-auth-logic

### PR DESCRIPTION
**Pull request by Pr0bka**

### CODE REVIEW

File: src/store/auth.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+/* eslint-disable no-param-reassign */
```
Comment (AntonOvc) [inline src/store/auth.js:1]: please create separate folder for state slices inside state folder

File: src/components/LoginForm/LoginForm.js [hunk 2]
```diff
@@ -10,16 +10,16 @@ import {
...
 import { PATHES } from '../../routes';
-import { AuthContext } from '../../context/Auth';
 import classes from './LoginForm.module.css';
 import Logo from '../../BabloLogo.png';
 import { validationProps } from '../../utils/validationForm';
 
 function InputForm() {
   const navigate = useNavigate();
-  const authContext = useContext(AuthContext);
+  const authContext = useSelector((state) => state.auth);
```
Comment (AntonOvc) [inline src/components/LoginForm/LoginForm.js:22]: use restructuring here `{ user } = useSelector`

**Review by AntonOvc — CHANGES_REQUESTED** [review]

File: src/store/auth.js (hunk 1 above)
Comment (Pr0bka) [reply src/store/auth.js]: fixed

File: src/components/LoginForm/LoginForm.js (hunk 2 above)
Comment (Pr0bka) [reply src/components/LoginForm/LoginForm.js]: fixed

**Review by AntonOvc — APPROVED** [review]

**Pull request merged**